### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by edunham**
This blog post has the following features:
n/a

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -6,3 +6,7 @@ This blog post has the following features:
 - [ ] A URL that's approved by Dev Advocacy
```
Comment (coltonbeach) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] A URL approved by Dev Advocacy
```

**Review by coltonbeach — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -3,6 +3,10 @@ This blog post has the following features:
 - [ ] A GitHub Repository with a polished README
 - [ ] A GitHub Repository under the github.com/oktadev account
 - [ ] A title that's approved by Dev Advocacy
-- [ ] A URL that's approved by Dev Advocacy
+- [ ] A URL approved by Dev Advocacy
 - [ ] The content has been run through Grammarly (https://www.grammarly.com/)
 - [ ] Rendered locally and confirmed that no Markdown typos exist
+- [ ] Images are compressed appropriately
+- [ ] Social image previews well in tweets
+- [ ] Review request for fact checking (developer advocate or domain expert)
```
Comment (alisaduncan) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Tech review request (developer advocate or domain expert)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -3,6 +3,10 @@ This blog post has the following features:
 - [ ] A GitHub Repository with a polished README
 - [ ] A GitHub Repository under the github.com/oktadev account
 - [ ] A title that's approved by Dev Advocacy
-- [ ] A URL that's approved by Dev Advocacy
+- [ ] A URL approved by Dev Advocacy
 - [ ] The content has been run through Grammarly (https://www.grammarly.com/)
 - [ ] Rendered locally and confirmed that no Markdown typos exist
+- [ ] Images are compressed appropriately
+- [ ] Social image previews well in tweets
```
Comment (alisaduncan) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Social image previews well on Twitter and LinkedIn
```

**Review by alisaduncan — COMMENTED** [review]
Changes requested

**Pull request merged**